docs/src/submodules/Bridges/overview.md

```diff
@@ -47,22 +52,30 @@ There are three types of bridges in MathOptInterface:
 ### Constraint bridges
 
 Constraint bridges convert constraints formulated by the user into an equivalent
-form supported by the solver.
+form supported by the solver. Constraint bridges are subtypes of 
+[`Bridges.Constraint.AbstractBridge`](@ref).
 
 The equivalent formulation may add constraints (and possibly also variables) in
 the underlying model.
 
+In particular, constraint bridges can focus on rewriting the function of a
+constraint, and do not change the set. Function bridges are subtypes of 
+[`Bridges.Constraint.AbstractFunctionConversionBridge`](@ref).
```

Comment:
Doc build failing because this docstring doesn't exist

Reply:
I think the problem is that this struct is not mentioned in the reference part of the Bridges submodule. Would https://github.com/dourouc05/MathOptInterface.jl/commit/451780afc58256f68eb032ef2942aa24caaacb7f be enough to solve this issue?

Reply:
I guess we'll see. But yeah, you just need to put it there.